apps/consulting/posts/evolving-qhub-to-nebari.md

```diff
@@ -0,0 +1,256 @@
+
+## How Far We Have Come, and What’s Next
+
+Nebari has come a long way since its [official launch in 2020]announcing qhub
+post! It currently supports all major cloud providers: Amazon Web Services
+(AWS), Google Cloud Provider (GCP), Azure, and Digital Ocean; as well as local
+HPC clusters.
+
+In preparation for the rename and change of governance, we spent the last few
+months focusing on improving the user and developer experiences. For example, we
+did a complete documentation re-write and re-structure. Nebari now follows the
+[Diátaxis][diataxis site] framework, and has a new website built using
+Docusaurus and hosted at [nebari.dev][nebari homepage].
+
+We updated Nebari’s command line interface (CLI) using Typer and Rich for a
+better user experience. We also created a new “Guided Init” wizard for new users
+to deploy Nebari more quickly and with less friction. You can learn more about
+these changes in the blog post
+[Developing a Typer CLI for Nebari][nebari typer cli post] by Asmi Jafar, a
+Quansight Labs intern, who worked on this feature.
+
+We improved integrations like Keycloak for better access and resource management
+and added new ones like Visual Studio Code and Grafana monitoring for
+convenience.
+
+<img
+src="/posts/evolving-qhub-to-nebari/6-nebari-help.png"
+width="850px"
+alt="Nebari's CLI showing the output of nebari --help command. The options, commands, and additional commands are displayed distinctly in separate boxes, with accessible colors to differentiate the command from their descriptions."
+/>
+
+In the coming months, we will continue to focus on Nebari’s community and
+governance documents. We want to add more guidelines for maintainers and more
+contributing paths that involve little to no code. We are actively improving and
+expanding the documentation, and making our integrations more robust.
+
+You can keep up with Nebari’s progress by keeping track of our
+[detailed roadmap][nebari governance repo] on GitHub!
+
+## Join Our Community!
+
+This milestone wouldn’t be possible without numerous contributors who worked on
+Nebari over the years, continued support from the Quansight team, and feedback
+from the broader open source community. We are immensely grateful to you!
+
+<img
+src="/posts/evolving-qhub-to-nebari/7-nebari-contributor-graph.png"
+width="875px"
+alt="GitHub contributor commit history for Nebari. It shows a steady flow of commits from March 2020 till November 2022. The activity in the first half of this period is bursty, while the second half is lower but more consistent."
+/>
+
+We’ve started hosting [bi-weekly meetings][nebari community meetings info] to
+give all community members, including users, contributors, and maintainers, a
+chance to collaborate and share ideas. We invite you to come by and say _hi_.
+
+Try out Nebari, and let us know how it goes. We look forward to seeing you in
+our community spaces!
+
+export const GoButton = () => {
```

Comment:
@gabalafou Can you help with this?

- This is probably really terrible React, cleanup welcomed
- For the life of me, I cannot find the Tailwind that will center this button in the blog text column
- I tried to import `Button` from `@quansight/shared/ui-components`, but it kept breaking the build
  - That said, I don't think we want exactly a `Button` here -- the padding on that shared componet is too wide on all four sides -- so this might not need to be solved
  - However, we likely do want to be able to import arbitrary components from the site codebase as a general capability, so figuring out how to do it here might be good

Reply:
Oh - I *had* it indented more nicely, but the auto-prettier in VS Code flattened it all to the left margin.

Reply:
Nevermind here, @gabalafou -- I figured out how to add new MDX-usable components (thanks for the README section there!), and I discovered that without .md and .mdx being defined in the Tailwind config, it wouldn't parse them for the classes used...and so it would minify out the styles I was trying to apply.

All fixed now.